HEC.FDA.Model/Spatial/StructureFactory.cs

```diff
@@ -0,0 +1,150 @@
+﻿using Geospatial.GDALAssist;
+using RasMapperLib;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using HEC.FDA.Model.utilities;
+using HEC.FDA.Model.structures;
+using Geospatial.Features;
+using Geospatial.IO;
+using Geospatial.Vectors;
+using Utility.Logging;
+using Geospatial.GDALAssist.Vectors;
+using Utility.Memory;
+using HEC.FDA.Model.Spatial.Extensions;
+
+namespace HEC.FDA.Model.Spatial;
+public class StructureFactory
```

Comment:
This is a stupid visualization of a class rename. This used to be ShapefileReader, which wasn't descriptive enough.